contracts/bonding/StakingManager.sol

```diff
@@ -28,39 +28,39 @@ contract StakingManager is ManagerProxyTarget, IStakingManager {
 
     struct Orchestrator {
         // Time-keeping
-        uint256 activationRound; // Round in which the transcoder became active - 0 if inactive
+        uint256 activationRound; // Round in which the orchestrator became active - 0 if inactive
         uint256 deactivationRound;
         // Commission accounting
         uint256 rewardShare; // % of reward shared with delegations
         uint256 feeShare; // % of fees shared with delegations
-        uint256 rewardCommissions; // reward earned from commission (not shared with delegators)
         uint256 feeCommissions; // fees earned from commission (not shared with delegators)
         uint256 lastRewardRound;
         // Delegation Pool
         Delegations.Pool delegationPool;
     }
 
-    // Represents an amount of tokens that are being unbonded
-    struct UnbondingLock {
+    // Represents an amount of tokens that are being undelegate
+    struct UnstakingLock {
+        address owner; // address which owns undelegated amount
```

Comment:
how about naming this "delegator" ? and changing the comment to something like "address of the delegator owning the unstaking lock"